### PR TITLE
Migrated gym to the new fastlane UI

### PR DESCRIPTION
**Pull request by KrauseFx**
Migrated gym to the new fastlane UI

### CODE REVIEW

File: spec/build_command_generator_spec.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
     it "raises an exception when project path wasn't found" do
       expect do
         Gym.config = FastlaneCore::Configuration.create(Gym::Options.available_options, { project: "/notExistent" })
-      end.to raise_error "Project file not found at path '/notExistent'".red
+      end.to raise_error "Project file not found at path '/notExistent'"
```
Comment (ohwutup) [inline spec/build_command_generator_spec.rb]: how come this doesn't use the new `UI` class?

**Comment by ohwutup** [comment]
this looks good to me! we should just make sure those ci jobs are green before merging :rocket:

File: spec/build_command_generator_spec.rb (hunk 1 above)
Comment (KrauseFx) [inline spec/build_command_generator_spec.rb]: This is a test, using the UI class throws standard exceptions underneath

**Comment by KrauseFx** [comment]
I don't like broken builds :cry:

**Pull request merged**